webapp/components/experiments/ExperimentEditStyle.js

```diff
@@ -0,0 +1,242 @@
+import {
+  bgDarkest,
+  bgLight,
+  bgRegular,
+  secondaryColor,
+  fontColor,
+  radius,
+  primaryColor,
+  experimentInputColor,
+  experimentFieldColor,
+  experimentSvgColor,
+  experimentLabelColor,
+  experimentAutocompleteBorder,
+  errorFieldBorder,
+} from '../../theme';
+
+/**
+ * Edit/Add view of a single Experiment.
+ *
+ * @return {JSX.Element}
+ * @constructor
+ */
+const useStyles = (theme) => ({
+  root: {
+    '& .editExperimentContainer': {
+      '& .editExperimentContent': {
+        overflow: 'auto',
+        maxHeight: 'calc(100vh - 400px)',
```

Comment:
Seems kind of arbitrary

Reply:
I just extracted existing code to a separate file, I agree with what you're saying